### PR TITLE
WIP: EEPROM Sync

### PR DESCRIPTION
**Pull request by sethp**
My First KiCad:

image

Some notes:

* The counter chips I had in my library were the wrong ones: I have the SN74HC163N chip, but the closest the library had was a 74LS163. The names for the pins are all a little different, but the numbers still line up so I think it's OK for now
* The N-bit counter chain can't be driven faster than 18MHz (see [datasheet](https://www.ti.com/lit/ds/symlink/sn74hc163.pdf?ts=1666729899859) ), but that's fine because we're only driving it at ~1.57MHz (1/16th of 25.175MHz)
* It seems like either I forgot a couple of important signals, or we've got 4 unused bits to play around with, or both
* I haven't actually built any part of this circuit yet, so who knows what goofy specters lurk within

Basically the only things I used to make this schematic were [step 2 of the tutorial](https://docs.kicad.org/6.0/en/getting_started_in_kicad/getting_started_in_kicad.html#tutorial_part_2_schematic) and then the ["bus connections" part of the eschema docs](https://docs.kicad.org/6.0/en/eeschema/eeschema.html#bus-connections).

### CODE REVIEW

**Comment by sethp** [comment]
After [eee2d7b](https://github.com/rustbox/cuddly-robot/pull/3/commits/eee2d7bc0a8ce3f362e0b1a412ecd04409bac912) the schematic looks like this: 

image

**Comment by sethp** [comment]
Actually, let's just keep working on this over in #4
